### ISSUE TITLE
Each video in the test set you provided only has the groundtruth of the first frame. Where can I find other groundtruths?

### ISSUE THREAD

**Issue by LxdBlack** [BasitAlawode/UVOT400#3]
Great job!
Each video in the test set you provided only has the groundtruth of the first frame. Where can I find other groundtruths?

**Comment by BasitAlawode** [comment]
Yes. This was deliberate and in line with best practices in object tracking and our plans for the dataset. The training set is available for use. An evaluation server will be set up soon to evaluate trackers on the test set. Lastly, a smaller version of our dataset (UTB180) is available for full evaluation. Thank you.
